.github/workflows/deploy.yaml

```diff
@@ -71,6 +71,8 @@ jobs:
     name: Deploy application to prod
     needs: [ test-build-and-push, deploy-dev ]
     runs-on: ubuntu-latest
+    permissions:
+      contents: "write"
```

Comment:
er du sikker på at det trengs når den har denne permissionen lenger opp i workflowen?

Reply:
Ja,https://github.com/navikt/amt-aktivitetskort-publisher/actions/runs/6046901665/job/16409486170,
Permissonen gjelder nok bare for en enkelt jobb

Reply:
det fungerte for denne ved deploy? https://github.com/navikt/amt-tiltaksarrangor-bff/pull/121

Reply:
Deploy til prod?

Reply:
hvis du synes pr-en min ser ok ut kan jeg teste ;)

Reply:
Hehe, ok, jeg testet i arena-ords-proxy og i aktivitetskort appen, så jeg bare kjørte oppdateringen i alle. For vi har lik workflow i alle backend apper så vidt jeg så.

Reply:
synes bare det er rart at det skulle virke for deploy til dev, men ikke for deploy til prod når jobbene er like bortsett fra miljøet 🤔  Men det kan hende du har rett

Reply:
Vi lager ikke release ved deploy til dev 😊

Reply:
ah, det er det den kicker på! da gor det mening :D

Reply:
Jeg kunne skrevet en bedre commit melding 😅

Reply:
jeg merget pr-en din for å fikse mine egne feil 😅